Dockerfile

```diff
@@ -48,7 +48,7 @@ RUN apt-get update \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/* \
     && ln -sf /usr/bin/llvm-symbolizer-14 /usr/bin/llvm-symbolizer \
-    && pip3 install --break-system-packages scipy pandas matplotlib  # preload large Python packages (installs numpy and
-                                                                       others).
+    && pip3 install --break-system-packages scipy pandas matplotlib  # preload large Python packages (installs numpy
```

Comment:
Can you please change line 9 to `ENV DEBIAN_FRONTEND=noninteractive`.

Just stumpled over the warning: ` - LegacyKeyValueFormat: "ENV key=value" should be used instead of legacy "ENV key value" format (line 9)`